tests/nest_tests/nest_custom_templates_test.py

```diff
@@ -18,19 +18,26 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
 import os
 import unittest
+import pytest
 
 import nest
 
+from pynestml.codegeneration.nest_tools import NESTTools
 from pynestml.frontend.pynestml_frontend import generate_target
 
+nest_version = NESTTools.detect_nest_version()
+
 
 class NestCustomTemplatesTest(unittest.TestCase):
     """
     Tests the code generation and installation with custom NESTML templates for NEST
     """
 
+    @pytest.mark.skipif(nest_version.startswith("v2"),
```

Comment:
Why don't these tests support NEST 2?

Reply:
No particular reason, except that the code of a lot of tests becomes considerably more ugly when adding NEST 2 support back in (``GetStatus()`` instead of ``get()`` and so on). So I restricted the set of NEST 2 tests to those testing the core functionality. We can adapt more tests on a case-by-case basis if we deem it necessary.